sor-api/src/main/java/com/bazaarvoice/emodb/sor/delta/impl/SetDeltaImpl.java

```diff
@@ -83,12 +85,13 @@ public void appendTo(Appendable buf) throws IOException {
         // Optimize the simple cases
         switch (literals.size()) {
             case 0:
-                return ImmutableSet.of();
+                return Collections.emptySet();
             case 1:
-                return ImmutableSet.of(literals.iterator().next());
+                return Collections.unmodifiableSet(new LinkedHashSet<>(literals));
         }
 
-        return ImmutableSortedSet.copyOf(literals);
+        Set<Literal> sortedSet = literals.stream().sorted().collect(Collectors.toCollection(TreeSet::new));
```

Comment:
Minor: Is `sorted()` necessary here since it's being collected into a `TreeSet` anyway?